content/astronomers/applications.md

```diff
@@ -0,0 +1,70 @@
+| **Catalogues visualisation:**  <br>[Aladin](http://aladin.u-strasbg.fr/aladin.gml), [TOPCAT](http://www.star.bris.ac.uk/%7Embt/topcat/), [IPAC Firefly](https://github.com/Caltech-IPAC/firefly) |
+| **Cross-correlation:**  <br>[Aladin](http://aladin.u-strasbg.fr/aladin.gml), [STILTs](http://www.star.bris.ac.uk/%7Embt/stilts/),[TOPCAT](http://www.star.bris.ac.uk/%7Embt/topcat/), [CDS Xmatch Service](http://cdsxmatch.u-strasbg.fr/xmatch) |
+| **Scatter, 3D plots and histograms:**  <br>[TOPCAT](http://www.star.bris.ac.uk/%7Embt/topcat/), [IPAC Firefly](https://github.com/Caltech-IPAC/firefly) |
+| **Statistics:**  <br>[AstroStat](http://voi.iucaa.in/voi/AstroStat.html) |
+| **Footprint Service:**  <br>[Aladin](http://aladin.u-strasbg.fr/aladin.gml), [VOServices](http://voservices.net/footprint/) |
+| **Table format conversion:**  <br>[TOPCAT](http://www.star.bris.ac.uk/%7Embt/topcat/), [STILTS](http://www.star.bris.ac.uk/%7Embt/stilts/) |
+| **Filter curves:**  <br>[VOServices](http://voservices.net/filter/), [Filter Profile Service](http://svo2.cab.inta-csic.es/theory/fps/) |
+| **SED building:**  <br>[VOSA](http://svo2.cab.inta-csic.es/theory/vosa/), [VOSpec](http://www.sciops.esa.int/index.php?project=SAT&page=vospec) |
+| **Fixing WCS:**  <br>[Aladin](http://aladin.u-strasbg.fr/aladin.gml) |
+| **Query Databases**  <br>[TOPCAT](http://www.star.bris.ac.uk/%7Embt/topcat/), [TAPHandle](http://saada.u-strasbg.fr/taphandle/), [IPAC Firefly](https://github.com/Caltech-IPAC/firefly) |
+
+| VO-compliant Tools & Services |
+| --- |
+| [DS9](http://hea-www.harvard.edu/RD/ds9/): Image visualisation |
+| [VirGO: Search for Images and Spectra](http://archive.eso.org/cms/tools-documentation/visual-archive-browser.html) |
+| [World Wide Telescope](http://www.worldwidetelescope.org/) |
+| [Gaia](http://star-www.dur.ac.uk/~pdraper/gaia/gaia.html) - Graphical Astronomy and Image Analysis |
+| [SIMBAD](http://simbad.u-strasbg.fr/simbad/) |
+| [TESELA](http://sdc.cab.inta-csic.es/tesela/index.jsp) |
+| [VizieR](http://vizier.u-strasbg.fr/viz-bin/VizieR) |
+|     |
+| Browse the Registries |
+| [WIRR - Web Interface to the Relational Registry](http://dc.zah.uni-heidelberg.de/wirr/q/ui/fixed) |
+| [RSS Feed of New & Updated Services in the VO](http://dc.zah.uni-heidelberg.de/registryrss/q/rss/info) |
+| [AppLauncher](http://www.jmmc.fr/applauncher) |
+| [EURO-VO Registry](https://registry.euro-vo.org/evor/) |
+
+_If you are interested in having your own project linked here, please [contact us](mailto:ivoadoc@ivoa.net)._
+
+### IVOA Applications Working Group
+
+Also visit the [Applications Working Group](http://wiki.ivoa.net/twiki/bin/view/IVOA/IvoaApplications) pages on the [IVOA](/) twiki, where you can find more [VO applications](http://wiki.ivoa.net/twiki/bin/view/IVOA/IvoaApplications#Links) in different levels of maturity.
```

Comment:
IVOA wiki should be `[IVOA wiki](https://wiki.ivoa.net/twiki/bin/view/IVOA)` otherwise the `(/)` simply links back to the IVOA web site.

Reply:
besides the link in the last md line, to me it looks fine and can be approved/merged